hack/tools.go

```diff
@@ -23,4 +23,7 @@ import (
 	_ "github.com/campoy/embedmd"
 	_ "github.com/jteeuwen/go-bindata/go-bindata"
 	_ "github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb"
+	_ "github.com/google/go-jsonnet/cmd/jsonnet"
+	_ "github.com/google/go-jsonnet/cmd/jsonnetfmt"
```

Comment:
I believe that you don't use `jsonnetfmt`

Reply:
Good find! We definitely should use it! I'll add it in a separate commit or PR.